scraper.js

```diff
@@ -0,0 +1,494 @@
+import request from "request";
+import cheerio from "cheerio";
+import db from "./helpers/db";
+
+let urls = [];
```

Comment:
Not sure why all of this code isn't just wrapped in db.connect

Reply:
Would it be better to have everything in a db.connect? what would be the benefit? how would one do such a thing?